crates/utils/src/error.rs

```diff
@@ -9,49 +10,185 @@ struct ApiError {
   error: String,
 }
 
+#[derive(EnumString, Display, Debug, Serialize, Deserialize, Clone)]
+#[cfg_attr(feature = "full", derive(TS))]
+#[cfg_attr(feature = "full", ts(export))]
+#[serde(tag = "error_type", content = "message")]
```

Comment:
You need to add `rename_all = "snake_case"` here. That should fix the api tests and avoid breaking changes.

https://serde.rs/container-attrs.html#rename_all